lib/clients/fake_rabbitmq.ex

```diff
@@ -44,6 +49,12 @@ defmodule ExRabbitPool.FakeRabbitMQ do
     {:ok, %Channel{conn: conn, pid: self()}}
   end
 
+  @impl true
+  def close_channel(_channel) do
+    # Channel.close(channel)
```

Comment:
Those commented lines, are necessary? the code style did not complain about it?

Reply:
Nah, we can delete those lines, they are just for me to see which method we are referencing, and no, the code style doesn't care about comments, i think those are omitted